src/zh/dmzj/src/eu/kanade/tachiyomi/extension/zh/dmzj/ApiSearch.kt

```diff
@@ -25,6 +25,33 @@ object ApiSearch {
         return MangasPage(data.map { it.toSManga() }, false)
     }
 
+    fun searchUrlV1(page: Int, query: String) =
```

Comment:
is the old search method used anywhere now?

Reply:
I remove the old search now.